public/robots.txt

```diff
@@ -3,6 +3,16 @@
 # To ban all spiders from the entire site uncomment the next two lines:
 # User-Agent: *
 # Disallow: /
+
+User-agent: SemrushBot
+Disallow: /
+
+User-agent: AhrefsBot
+Disallow: /
```

Comment:
According to [Wikipedia](https://en.wikipedia.org/wiki/Robots_exclusion_standard#Examples) you can shorten that to:

```
User-agent: SemrushBot
User-agent: AhrefsBot
Disallow: /
```